tcl/dialog_iemgui.tcl

```diff
@@ -12,6 +12,24 @@ namespace eval ::dialog_iemgui:: {
     namespace export pdtk_iemgui_dialog
 }
 
+source [file join [file dirname [info script]] "dialog_iemgui_knob.tcl"]
+
+proc procExists p {
```

Comment:
Can this be snake case the match the rest of the API?

Reply:
sorry I don't understand "snake case"... could you help me?

Reply:
OK! you mean using `proc_exists`

Reply:
done